.github/CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+  
+# Maps code in repository with maintainers
+# Order is important. The last matching pattern has the most precedence.
+# SEE https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file
+
+
+# files and folders recursively
+Dockerfile          @elisabettai
+Makefile            @elisabettai
+
+# NOTE: '/' denotes the root of the repository
+/.github/            @elisabettai
+/.osparc/            @elisabettai
```

Comment:
I would like to suggest to share ownership with @cbujard

Reply:
Good point, I added him,.

Reply:
where?

Reply:
Ops, forgot to save the file, thanks! I've now added him to the CODEOWNERS (and gave him write access to the repo).